app/src/main/java/com/abhijitvalluri/android/fitnotifications/AppSettingsActivity.java

```diff
@@ -186,16 +190,18 @@ public void onBackPressed() {
     public void onActivityResult2(int requestCode, Intent data) {
         int hour = data.getIntExtra(TimePickerFragment.EXTRA_HOUR, 0);
         int minute = data.getIntExtra(TimePickerFragment.EXTRA_MINUTE, 0);
-        if (requestCode == START_TIME_REQUEST) {
+
+        java.text.DateFormat timeFormat = DateFormat.getTimeFormat(this);
+        String formattedTime = timeFormat.format(Func.convertHourMinute2Date(hour, minute));
+
+        if (requestCode == START_TIME_REQUEST) {`
```

Comment:
Sorry, this is what happens when switching between three different branches trying to maintain diverged project files.
By the way, why did you add *.iml to `.gitignore`? Having modules under version control would allow maintaining branch-specific project state much easier.